go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/streamingfast/firehose-core
 
-go 1.21
+go 1.22
```

Comment:
We won't be able to release using `sfreleaser` for now as the `goreleaser-cross` Docker image with Go 1.22 is not published yet (although 3 days ago a PR was merged to bring it in, so it seems to be coming).

Reply:
hmm ok I'll try to release it under 1.21 for now then !!!